src/market/scan/scanned-offer.ts

```diff
@@ -51,34 +56,104 @@ export class ScannedOffer {
       name: this.properties["golem.node.id.name"] || "<unknown>",
     };
   }
+
   get transferProtocol() {
     return this.properties["golem.activity.caps.transfer.protocol"];
   }
+
   get cpuBrand() {
     return this.properties["golem.inf.cpu.brand"];
   }
+
+  get cpuVendor() {
+    return this.properties["golem.inf.cpu.vendor"];
+  }
+
   get cpuCapabilities() {
     return this.properties["golem.inf.cpu.capabilities"];
   }
+
   get cpuCores() {
     return this.properties["golem.inf.cpu.cores"];
   }
+
   get cpuThreads() {
     return this.properties["golem.inf.cpu.threads"];
   }
+
+  get gpuBrand() {
+    return this.properties["golem.!exp.gap-35.v1.inf.gpu.model"];
+  }
+
+  /** @deprecated Use {@link memoryGib} instead */
   get memory() {
+    return this.memoryGib;
+  }
+
+  get memoryGib() {
     return this.properties["golem.inf.mem.gib"];
   }
+
+  /** @deprecated Use {@link storageGib} instead */
   get storage() {
+    return this.storageGib;
+  }
+
+  get storageGib() {
     return this.properties["golem.inf.storage.gib"];
   }
+
   get publicNet() {
     return this.properties["golem.node.net.is-public"];
   }
+
   get runtimeCapabilities() {
     return this.properties["golem.runtime.capabilities"];
   }
+
   get runtimeName() {
     return this.properties["golem.runtime.name"];
   }
+
+  /**
+   * Get the ID of the offer published by the Provider
+   *
+   * Note:
+   * - this ID will change after the provider refreshes the offer (usually after 1h)
+   * - this ID will remain unchanged for the same published offer between different scans
+   */
+  get offerId() {
+    return this.model.offerId;
+  }
+
+  /**
+   * The timestamp at which the offer was generated by the Provider
+   */
+  get offerCreateAt() {
+    return this.model.timestamp;
+  }
+
+  /**
+   * Lists down payment addresses on different payment platforms
+   *
+   * @example Example return value
+   * ```json
+   * {
+   *  "erc20-polygon-glm": "0x8737beea5668595fda9d50e85cae9cad10b4c980",
+   *  "erc20-holesky-tglm:" "0x8737beea5668595fda9d50e85cae9cad10b4c980",
+   * }
+   * ```
+   */
+  get paymentPlatformAddresses(): PaymentPlatformAddressSet {
+    const platformProps = Object.entries(this.model.properties).filter(([key]) =>
+      key.startsWith("golem.com.payment.platform."),
+    );
+
+    const platformAddress = platformProps.map(([key, address]) => [
+      key.replace("golem.com.payment.platform.", "").replace(".address", ""),
```

Comment:
if someone sets a constraint that starts with `golem.com.payment.platform` but ends with something different from `.<platform>.address` then this method will produce nonsense. We should filter the properties not only with `startsWith` but also with `endsWith` or even better, with a single regex like
`/^golem\.com\.payment\.platform\.[a-zA-Z\-0-9]+\.address$/` (untested)